|===
|Path|Type|Description

|`+age_entrez_acc_id+`
|`+Number+`
|The Entrez accession ID related to age studies.

|`+age_human_gene_id+`
|`+Number+`
|The human gene ID associated with age studies.

|`+age_id+`
|`+Number+`
|Unique identifier for the age-related study entry.

|`+age_mean_gene_effect+`
|`+Number+`
|The mean gene effect observed in the age study.

|`+age_raw_data_id+`
|`+Number+`
|Raw data identifier associated with age studies.

|`+cav_allele_accession_id+`
|`+String+`
|Allele accession ID in CAV repository.

|`+cav_allele_symbol+`
|`+String+`
|Symbol of the allele in the CAV repository.

|`+cav_category+`
|`+Array+`
|Categories associated with the CAV allele.

|`+clin_date+`
|`+String+`
|Date associated with clinical data entry.

|`+clin_haploinsufficiency+`
|`+String+`
|Clinical haploinsufficiency score.

|`+clin_human_gene_id+`
|`+Number+`
|Human gene ID related to clinical data.

|`+clin_id+`
|`+Number+`
|Unique identifier for the clinical data entry.

|`+clin_report+`
|`+String+`
|Textual report associated with clinical data.

|`+clin_triplosensitivity+`
|`+String+`
|Clinical triplosensitivity score.

|`+f_bin+`
|`+String+`
|Functional binning data.

|`+f_bin_code+`
|`+String+`
|Code representing a functional bin.

|`+f_id+`
|`+Number+`
|Unique identifier for functional data entry.

|`+f_ortholog_id+`
|`+Number+`
|Ortholog ID associated with functional data.

|`+gnp_brain_expression+`
|`+String+`
|Genomic data pertaining to brain expression.

|`+gnp_cds_length+`
|`+Number+`
|Length of the coding sequence in genomic data.

|`+gnp_chromosome+`
|`+String+`
|Chromosome location in genomic data.

|`+gnp_classic_caf+`
|`+Number+`
|General Combined Annotation Dependent Depletion score for all populations.

|`+gnp_classic_caf_afr+`
|`+Number+`
|CADD score for African population.

|`+gnp_classic_caf_amr+`
|`+Number+`
|CADD score for American population.

|`+gnp_classic_caf_asj+`
|`+Number+`
|CADD score for Ashkenazi Jewish population.

|`+gnp_classic_caf_eas+`
|`+Number+`
|CADD score for East Asian population.

|`+gnp_classic_caf_fin+`
|`+Number+`
|CADD score for Finnish population.

|`+gnp_classic_caf_nfe+`
|`+Number+`
|CADD score for Non-Finnish European population.

|`+gnp_classic_caf_oth+`
|`+Number+`
|CADD score for Other populations.

|`+gnp_classic_caf_sas+`
|`+Number+`
|CADD score for South Asian population.

|`+gnp_constraint_flag+`
|`+String+`
|Flag indicating genomic constraint.

|`+gnp_defined+`
|`+Number+`
|Indicator if genomic data is defined.

|`+gnp_end_position+`
|`+Number+`
|End position of the gene in the genome.

|`+gnp_exac_exp_lof+`
|`+Number+`
|Expected loss of function based on ExAC database.

|`+gnp_exac_obs_lof+`
|`+Number+`
|Observed loss of function based on ExAC database.

|`+gnp_exac_oe_lof+`
|`+Number+`
|Observed/Expected loss of function ratio based on ExAC database.

|`+gnp_exac_pli+`
|`+Number+`
|Probability of being loss-of-function intolerant (pLI) based on ExAC database.

|`+gnp_exp_hom_lof+`
|`+Number+`
|Expected homozygous loss of function.

|`+gnp_exp_lof+`
|`+Number+`
|Expected loss of function.

|`+gnp_exp_mis+`
|`+Number+`
|Expected missense mutation.

|`+gnp_exp_mis_pphen+`
|`+Number+`
|Expected missense mutation with probably damaging phenotype.

|`+gnp_exp_syn+`
|`+Number+`
|Expected synonymous mutation.

|`+gnp_gene_id+`
|`+String+`
|Gene ID in the Genomics England PanelApp.

|`+gnp_gene_length+`
|`+Number+`
|Length of the gene in the genome.

|`+gnp_gene_symbol+`
|`+String+`
|Symbol of the gene in the genomic data.

|`+gnp_gene_type+`
|`+String+`
|Type of the gene in genomic data.

|`+gnp_human_gene_id+`
|`+Number+`
|Human gene ID associated with the genomic data.

|`+gnp_id+`
|`+Number+`
|Unique identifier for the genomic data entry.

|`+gnp_lof_z+`
|`+Number+`
|Z-score for the loss of function (LoF) variant.

|`+gnp_max_af+`
|`+Number+`
|Maximum allele frequency across all populations.

|`+gnp_mis_z+`
|`+Number+`
|Z-score for missense variants.

|`+gnp_mu_lof+`
|`+Number+`
|Mutation rate for loss of function.

|`+gnp_mu_mis+`
|`+Number+`
|Mutation rate for missense variants.

|`+gnp_mu_syn+`
|`+Number+`
|Mutation rate for synonymous variants.

|`+gnp_n_sites+`
|`+Number+`
|Number of sites evaluated.

|`+gnp_no_lofs+`
|`+Number+`
|Number of observed loss-of-function variants.

|`+gnp_num_coding_exons+`
|`+Number+`
|Number of coding exons.

|`+gnp_obs_het_lof+`
|`+Number+`
|Observed heterozygous loss of function variants.

|`+gnp_obs_hom_lof+`
|`+Number+`
|Observed homozygous loss of function variants.

|`+gnp_obs_lof+`
|`+Number+`
|Total observed loss of function variants.

|`+gnp_obs_mis+`
|`+Number+`
|Observed missense variants.

|`+gnp_obs_mis_pphen+`
|`+Number+`
|Observed missense variants predicted to be probably damaging.

|`+gnp_obs_syn+`
|`+Number+`
|Observed synonymous variants.

|`+gnp_oe_lof+`
|`+Number+`
|Observed/Expected ratio for loss of function variants.

|`+gnp_oe_lof_lower+`
|`+Number+`
|Lower bound of the 95% confidence interval for the O/E ratio for LoF variants.

|`+gnp_oe_lof_upper+`
|`+Number+`
|Upper bound of the 95% confidence interval for the O/E ratio for LoF variants.

|`+gnp_oe_lof_upper_bin+`
|`+Number+`
|Bin value for the upper bound of O/E ratio for LoF variants.

|`+gnp_oe_lof_upper_bin6+`
|`+Number+`
|Bin value for the upper bound of O/E ratio for LoF variants, using a different binning strategy.

|`+gnp_oe_lof_upper_rank+`
|`+Number+`
|Rank of the gene based on the upper bound of O/E ratio for LoF variants.

|`+gnp_oe_mis+`
|`+Number+`
|Observed/Expected ratio for missense variants.

|`+gnp_oe_mis_lower+`
|`+Number+`
|Lower bound of the 95% confidence interval for the O/E ratio for missense variants.

|`+gnp_oe_mis_pphen+`
|`+Number+`
|Observed/Expected ratio for probably damaging missense variants.

|`+gnp_oe_mis_upper+`
|`+Number+`
|Upper bound of the 95% confidence interval for the O/E ratio for missense variants.

|`+gnp_oe_syn+`
|`+Number+`
|Observed/Expected ratio for synonymous variants.

|`+gnp_oe_syn_lower+`
|`+Number+`
|Lower bound of the 95% confidence interval for the O/E ratio for synonymous variants.

|`+gnp_oe_syn_upper+`
|`+Number+`
|Upper bound of the 95% confidence interval for the O/E ratio for synonymous variants.

|`+gnp_p+`
|`+Number+`
|P-value for statistical tests.

|`+gnp_p_afr+`
|`+Number+`
|P-value for the African population.

|`+gnp_p_amr+`
|`+Number+`
|P-value for the American population.

|`+gnp_p_asj+`
|`+Number+`
|P-value for the Ashkenazi Jewish population.

|`+gnp_p_eas+`
|`+Number+`
|P-value for the East Asian population.

|`+gnp_p_fin+`
|`+Number+`
|P-value for the Finnish population.

|`+gnp_p_nfe+`
|`+Number+`
|P-value for the Non-Finnish European population.

|`+gnp_p_oth+`
|`+Number+`
|P-value for other populations.

|`+gnp_p_sas+`
|`+Number+`
|P-value for the South Asian population.

|`+gnp_pli+`
|`+Number+`
|pLI score indicating the probability of being loss-of-function intolerant.

|`+gnp_pnull+`
|`+Number+`
|Probability of being functionally neutral.

|`+gnp_possible_lof+`
|`+Number+`
|Possible loss-of-function variants.

|`+gnp_possible_mis+`
|`+Number+`
|Possible missense variants.

|`+gnp_possible_mis_pphen+`
|`+Number+`
|Possible probably damaging missense variants.

|`+gnp_possible_syn+`
|`+Number+`
|Possible synonymous variants.

|`+gnp_prec+`
|`+Number+`
|Precision of the genomic data.

|`+gnp_start_position+`
|`+Number+`
|Start position of the gene in the genome.

|`+gnp_syn_z+`
|`+Number+`
|Z-score for the synonymous variants.

|`+gnp_transcript+`
|`+String+`
|Transcript ID associated with the genomic data.

|`+gnp_transcript_level+`
|`+Number+`
|Level of the transcript associated with the genomic data.

|`+gnp_transcript_type+`
|`+String+`
|Type of transcript recorded in the genomic data.

|`+hg_ensembl_gene_acc_id+`
|`+String+`
|Ensembl gene accession ID for the human gene.

|`+hg_entrez_gene_acc_id+`
|`+Number+`
|Entrez gene accession ID for the human gene.

|`+hg_hgnc_acc_id+`
|`+String+`
|HGNC accession ID for the human gene.

|`+hg_id+`
|`+Number+`
|Unique identifier for the human gene entry.

|`+hg_name+`
|`+String+`
|Name of the human gene.

|`+hg_symbol+`
|`+String+`
|Symbol representing the human gene.

|`+hgnc_agr_acc_id+`
|`+String+`
|Agronomy accession ID related to the human gene.

|`+hgnc_alias_name+`
|`+String+`
|Alias name(s) for the human gene.

|`+hgnc_alias_symbol+`
|`+String+`
|Alias symbol(s) for the human gene.

|`+hgnc_bioparadigms_slc+`
|`+String+`
|Bioparadigms SLC code for the human gene.

|`+hgnc_ccds_acc_id+`
|`+String+`
|CCDS accession ID for the human gene.

|`+hgnc_cd+`
|`+String+`
|CD marker for the human gene.

|`+hgnc_cosmic+`
|`+String+`
|COSMIC ID for the human gene.

|`+hgnc_date_approved_reserved+`
|`+String+`
|Date the gene was approved or reserved in HGNC.

|`+hgnc_date_modified+`
|`+String+`
|Date the gene information was last modified in HGNC.

|`+hgnc_date_name_changed+`
|`+String+`
|Date the gene name was changed in HGNC.

|`+hgnc_date_symbol_changed+`
|`+String+`
|Date the gene symbol was changed in HGNC.

|`+hgnc_ena+`
|`+String+`
|European Nucleotide Archive ID for the human gene.

|`+hgnc_ensembl_gene_acc_id+`
|`+String+`
|Ensembl gene accession ID associated with the HGNC entry.

|`+hgnc_entrez_acc_id+`
|`+Number+`
|Entrez accession ID associated with the HGNC entry.

|`+hgnc_enzyme_acc_id+`
|`+String+`
|Enzyme accession ID associated with the human gene.

|`+hgnc_gencc+`
|`+String+`
|GenCC ID for the human gene.

|`+hgnc_gene_group+`
|`+String+`
|Gene group classification in HGNC.

|`+hgnc_gene_group_acc_id+`
|`+String+`
|Gene group accession ID in HGNC.

|`+hgnc_gtrnadb+`
|`+String+`
|tRNA database ID for the human gene.

|`+hgnc_hgnc_acc_id+`
|`+String+`
|HGNC accession ID for the human gene.

|`+hgnc_homeodb+`
|`+Number+`
|Homeobox database ID for the human gene.

|`+hgnc_horde_acc_id+`
|`+String+`
|Horde ID for the human gene (related to olfactory receptors).

|`+hgnc_human_gene_id+`
|`+Number+`
|Human gene ID associated with HGNC.

|`+hgnc_id+`
|`+Number+`
|Unique identifier for the HGNC entry.

|`+hgnc_imgt+`
|`+String+`
|Immunogenetics (IMGT) ID for the human gene.

|`+hgnc_intermediate_filament_db+`
|`+String+`
|Intermediate filament database ID for the human gene.

|`+hgnc_iuphar+`
|`+String+`
|International Union of Pharmacology (IUPHAR) ID for the human gene.

|`+hgnc_kznf_gene_catalog+`
|`+Number+`
|KZNF gene catalog ID for the human gene.

|`+hgnc_lncipedia+`
|`+String+`
|Lncipedia ID for the human gene.

|`+hgnc_lncrnadb+`
|`+String+`
|lncRNA database ID for the human gene.

|`+hgnc_location+`
|`+String+`
|Chromosomal location of the human gene.

|`+hgnc_location_sortable+`
|`+String+`
|Sortable format of the chromosomal location of the human gene.

|`+hgnc_locus_group+`
|`+String+`
|Locus group classification of the human gene.

|`+hgnc_locus_type+`
|`+String+`
|Locus type classification of the human gene.

|`+hgnc_lsdb+`
|`+String+`
|Locus-specific database ID for the human gene.

|`+hgnc_mamit_trnadb+`
|`+Number+`
|Mamit-tRNA database ID for the human gene.

|`+hgnc_mane_select+`
|`+String+`
|MANE Select ID for the human gene.

|`+hgnc_merops+`
|`+String+`
|MEROPS database ID for the human gene related to proteases.

|`+hgnc_mgi_gene_acc_id+`
|`+String+`
|MGI gene accession ID for the mouse ortholog of the human gene.

|`+hgnc_mirbase+`
|`+String+`
|miRBase ID for the human gene.

|`+hgnc_name+`
|`+String+`
|Official name of the human gene.

|`+hgnc_omim_acc_id+`
|`+String+`
|OMIM accession ID for the human gene, linking to the Online Mendelian Inheritance in Man database.

|`+hgnc_orphanet+`
|`+Number+`
|Orphanet ID for the human gene, relevant for rare diseases.

|`+hgnc_prev_name+`
|`+String+`
|Previous name(s) of the human gene.

|`+hgnc_prev_symbol+`
|`+String+`
|Previous symbol(s) of the human gene.

|`+hgnc_pseudogene_org+`
|`+String+`
|Pseudogene.org identifier for the human gene if applicable.

|`+hgnc_pubmed_acc_id+`
|`+String+`
|PubMed accession ID(s) associated with the human gene.

|`+hgnc_refseq_accession+`
|`+String+`
|RefSeq accession number for the human gene.

|`+hgnc_rgd_acc_id+`
|`+String+`
|Rat Genome Database (RGD) ID for the rat ortholog of the human gene.

|`+hgnc_rna_central_acc_ids+`
|`+String+`
|RNAcentral IDs for the human gene, representing non-coding RNA sequences.

|`+hgnc_snornabase+`
|`+String+`
|snoRNABase ID for the human gene if it encodes a small nucleolar RNA (snoRNA).

|`+hgnc_status+`
|`+String+`
|The status of the human gene in the HGNC database.

|`+hgnc_symbol+`
|`+String+`
|The official symbol of the human gene as designated by HGNC.

|`+hgnc_ucsc_acc_id+`
|`+String+`
|UCSC Genome Browser accession ID for the human gene.

|`+hgnc_uniprot_acc_ids+`
|`+String+`
|UniProt accession IDs for the protein products of the human gene.

|`+hgnc_vega_acc_id+`
|`+String+`
|VEGA database ID for the human gene.

|`+hgs_synonyms+`
|`+Array+`
|List of synonyms for the human gene symbol.

|`+id+`
|`+String+`
|The primary identifier for the essential gene entry.

|`+idg_family+`
|`+String+`
|The Integrative Genomics Viewer (IDG) family the gene belongs to.

|`+idg_human_gene_id+`
|`+Number+`
|The human gene ID according to IDG.

|`+idg_id+`
|`+Number+`
|The IDG unique identifier for the gene.

|`+mg_ensembl_chromosome+`
|`+String+`
|The chromosome number from the Ensembl database for the mouse gene.

|`+mg_ensembl_gene_acc_id+`
|`+String+`
|Ensembl gene accession ID for the mouse gene.

|`+mg_ensembl_start+`
|`+Number+`
|Start position of the mouse gene in the Ensembl database.

|`+mg_ensembl_stop+`
|`+Number+`
|Stop position of the mouse gene in the Ensembl database.

|`+mg_ensembl_strand+`
|`+String+`
|Strand orientation of the mouse gene in the Ensembl database.

|`+mg_entrez_gene_acc_id+`
|`+Number+`
|Entrez gene accession ID for the mouse gene.

|`+mg_genome_build+`
|`+String+`
|The genome build used for the mouse gene.

|`+mg_id+`
|`+Number+`
|Unique identifier for the mouse gene entry.

|`+mg_mgi_chromosome+`
|`+String+`
|MGI chromosome number for the mouse gene.

|`+mg_mgi_cm+`
|`+String+`
|Centimorgan position on the MGI database for the mouse gene.

|`+mg_mgi_gene_acc_id+`
|`+String+`
|MGI gene accession ID for the mouse gene.

|`+mg_mgi_start+`
|`+Number+`
|Start position of the mouse gene in the MGI database.

|`+mg_mgi_stop+`
|`+Number+`
|Stop position of the mouse gene in the MGI database.

|`+mg_mgi_strand+`
|`+String+`
|Strand orientation of the mouse gene in the MGI database.

|`+mg_name+`
|`+String+`
|Name of the mouse gene.

|`+mg_ncbi_chromosome+`
|`+String+`
|NCBI chromosome location for the mouse gene.

|`+mg_ncbi_start+`
|`+Number+`
|NCBI start position of the mouse gene.

|`+mg_ncbi_stop+`
|`+Number+`
|NCBI stop position of the mouse gene.

|`+mg_ncbi_strand+`
|`+String+`
|NCBI strand orientation of the mouse gene.

|`+mg_subtype+`
|`+String+`
|Subtype classification of the mouse gene.

|`+mg_symbol+`
|`+String+`
|Symbol representing the mouse gene.

|`+mg_type+`
|`+String+`
|Type classification of the mouse gene.

|`+mgs_synonyms+`
|`+Array+`
|List of synonyms for the mouse gene.

|`+o_category+`
|`+String+`
|Orthology category for gene comparison.

|`+o_human_gene_id+`
|`+Number+`
|Human gene ID used in ortholog comparison.

|`+o_id+`
|`+Number+`
|Unique identifier for the orthology data entry.

|`+o_is_max_human_to_mouse+`
|`+String+`
|Indicates if this is the maximum score from human to mouse.

|`+o_is_max_mouse_to_human+`
|`+String+`
|Indicates if this is the maximum score from mouse to human.

|`+o_mouse_gene_id+`
|`+Number+`
|Mouse gene ID used in ortholog comparison.

|`+o_support+`
|`+String+`
|Support level for the orthology assertion, indicating the strength of evidence.

|`+o_support_count+`
|`+Number+`
|Numerical support count for the orthology assertion, quantifying the evidence.

|`+o_support_raw+`
|`+String+`
|Raw support data for the orthology assertion, providing detailed evidence.

|`+pharos_chr+`
|`+String+`
|Pharos chromosome information for the human gene.

|`+pharos_family+`
|`+String+`
|Pharos family classification of the human gene.

|`+pharos_human_gene_id+`
|`+Number+`
|Human gene ID according to Pharos.

|`+pharos_id+`
|`+Number+`
|Unique identifier for the Pharos data entry.

|`+pharos_name+`
|`+String+`
|Gene name as listed in Pharos.

|`+pharos_symbol+`
|`+String+`
|Gene symbol as listed in Pharos.

|`+pharos_tdl+`
|`+String+`
|Target Development Level in Pharos, indicating the gene's research and development status.

|`+pharos_uniprot_acc_id+`
|`+String+`
|UniProt accession ID as recorded in Pharos for the human gene.

|===