|===
|Path|Type|Description

|`+mgi_accession_id+`
|`+String+`
|The Mouse Genome Informatics (MGI) accession ID uniquely identifying the gene.

|`+data_type+`
|`+String+`
|The type of data, indicating the nature of the information (e.g., 'gene', 'allele').

|`+marker_type+`
|`+String+`
|The type of genetic marker, such as 'protein coding gene', 'non-coding RNA'.

|`+marker_symbol+`
|`+String+`
|The official symbol of the genetic marker.

|`+marker_synonym+`
|`+Array+`
|A list of alternative names or symbols for the genetic marker.

|`+marker_name+`
|`+String+`
|The full name of the genetic marker.

|`+human_gene_symbol+`
|`+Array+`
|A list of human gene symbols that are orthologous to the mouse gene.

|`+ensembl_gene_id+`
|`+Array+`
|A list of Ensembl gene IDs associated with the marker.

|`+imits_phenotype_started+`
|`+String+`
|Indicates whether phenotyping has started for the marker ('true' or 'false').

|`+imits_phenotype_complete+`
|`+String+`
|Indicates whether phenotyping has been completed for the marker ('true' or 'false').

|`+imits_phenotype_status+`
|`+String+`
|The current status of the phenotyping effort.

|`+status+`
|`+String+`
|The current status of the gene or allele.

|`+latest_production_status+`
|`+String+`
|The most recent production status of the gene or allele.

|`+gene_latest_es_cell_status+`
|`+String+`
|The latest status of ES cell production for the gene.

|`+latest_es_cell_status+`
|`+String+`
|The latest status of ES cell production.

|`+gene_latest_mouse_status+`
|`+String+`
|The latest status of mouse production for the gene.

|`+latest_mouse_status+`
|`+String+`
|The latest status of mouse production.

|`+legacy_phenotype_status+`
|`+Number+`
|A status indicating the legacy phenotype information.

|`+latest_phenotype_status+`
|`+String+`
|The most recent phenotype status.

|`+latest_project_status+`
|`+String+`
|The most recent status of the project involving the gene or allele.

|`+latest_production_centre+`
|`+Array+`
|A list of centers where the latest production occurred.

|`+latest_phenotyping_centre+`
|`+Array+`
|A list of centers where the latest phenotyping occurred.

|`+chr_name+`
|`+String+`
|The name of the chromosome where the marker is located.

|`+chr_start+`
|`+Number+`
|The start position of the marker on the chromosome.

|`+chr_end+`
|`+Number+`
|The end position of the marker on the chromosome.

|`+chr_strand+`
|`+String+`
|The strand of the chromosome where the marker is located ('+' or '-').

|`+allele_name+`
|`+Array+`
|A list of names for alleles associated with the gene.

|`+allele_accession_id+`
|`+Array+`
|A list of accession IDs for alleles associated with the gene.

|`+es_cell_status+`
|`+Array+`
|A list of statuses for ES cell production for the gene.

|`+imits_mouse_status+`
|`+String+`
|The IMPC status of mouse production.

|`+mouse_status+`
|`+Array+`
|A list of statuses for mouse production.

|`+phenotype_status+`
|`+Array+`
|A list of phenotype statuses for the gene or allele.

|`+production_centre+`
|`+Array+`
|A list of centers involved in the production of the gene or allele.

|`+phenotyping_centre+`
|`+Array+`
|A list of centers involved in the phenotyping of the gene or allele.

|`+go_term_id+`
|`+Array+`
|A list of Gene Ontology (GO) term IDs associated with the gene.

|`+go_term_name+`
|`+Array+`
|A list of names for the Gene Ontology (GO) terms associated with the gene.

|`+go_term_def+`
|`+Array+`
|A list of definitions for the Gene Ontology (GO) terms associated with the gene.

|`+go_term_evid+`
|`+Array+`
|A list of evidence codes for the Gene Ontology (GO) terms associated with the gene.

|`+go_term_domain+`
|`+Array+`
|A list of domains (e.g., biological process, cellular component) for the GO terms associated with the gene.

|`+go_count+`
|`+Number+`
|The count of Gene Ontology (GO) terms associated with the gene.

|`+go_uniprot+`
|`+Array+`
|A list of UniProt identifiers linked to the GO terms.

|`+evid_code_rank+`
|`+Number+`
|A ranking of the evidence code for the Gene Ontology (GO) terms, indicating the strength of the evidence.

|`+pfama_json+`
|`+Array+`
|A JSON string or list of strings containing Pfam-A domain information.

|`+scdb_id+`
|`+Array+`
|A list of secondary database IDs related to the gene, providing external references.

|`+scdb_link+`
|`+Array+`
|A list of links to secondary databases that provide additional information about the gene.

|`+clan_id+`
|`+Array+`
|A list of Pfam clan IDs associated with the gene, grouping similar protein families.

|`+clan_acc+`
|`+Array+`
|A list of accession numbers for Pfam clans associated with the gene.

|`+clan_desc+`
|`+Array+`
|A list of descriptions for the Pfam clans associated with the gene.

|`+pfama_id+`
|`+Array+`
|A list of Pfam-A IDs associated with the gene, identifying protein families.

|`+pfama_acc+`
|`+Array+`
|A list of accession numbers for Pfam-A families associated with the gene.

|`+pfama_go_id+`
|`+Array+`
|A list of Gene Ontology (GO) IDs related to Pfam-A families.

|`+pfama_go_term+`
|`+Array+`
|A list of Gene Ontology (GO) terms related to Pfam-A families.

|`+pfama_go_cat+`
|`+Array+`
|A list of categories for the GO terms related to Pfam-A families.

|`+gf_acc+`
|`+String+`
|A unique accession ID for the gene family to which the gene belongs.

|`+allele_mgi_accession_id+`
|`+String+`
|The MGI accession ID specifically for the allele of the gene.

|`+imits_es_cell_status+`
|`+Array+`
|A list of statuses regarding the production of ES cells for the allele.

|`+ikmc_project+`
|`+String+`
|The ID of the International Knockout Mouse Consortium project associated with the gene.

|`+gene_allele+`
|`+String+`
|The specific allele of the gene.

|`+disease_human_phenotypes+`
|`+Array+`
|A list of human phenotypes associated with the gene.

|`+type+`
|`+String+`
|The type of entity described, such as 'gene', 'allele', or 'disease'.

|`+disease_id+`
|`+Array+`
|A list of IDs for diseases associated with the gene.

|`+disease_source+`
|`+Array+`
|A list of sources providing information about the diseases associated with the gene.

|`+disease_term+`
|`+Array+`
|A list of disease terms or names associated with the gene.

|`+disease_alts+`
|`+Array+`
|A list of alternative names or synonyms for the diseases associated with the gene.

|`+disease_classes+`
|`+Array+`
|A list of classifications for the diseases associated with the gene.

|`+human_curated+`
|`+Array+`
|Indicators of whether the disease association was curated by human experts.

|`+mouse_curated+`
|`+Array+`
|Indicators of whether the disease association was curated based on mouse models.

|`+mgi_predicted+`
|`+Array+`
|Indicators of whether the disease association is predicted by MGI.

|`+impc_predicted+`
|`+Array+`
|Indicators of whether the disease association is predicted by IMPC.

|`+mgi_predicted_known_gene+`
|`+Array+`
|Indicators of whether the known gene's disease association is predicted by MGI.

|`+impc_predicted_known_gene+`
|`+Array+`
|Indicators of whether the known gene's disease association is predicted by IMPC.

|`+mgi_novel_predicted_in_locus+`
|`+Array+`
|Indicators of whether there is a novel prediction in the locus by MGI.

|`+impc_novel_predicted_in_locus+`
|`+Array+`
|Indicators of whether there is a novel prediction in the locus by IMPC.

|`+marker_symbol_bf+`
|`+Array+`
|A list of marker symbols used for Bloom filter indexing.

|`+marker_symbol_lowercase+`
|`+Array+`
|A list of marker symbols in lowercase for case-insensitive searches.

|`+marker_synonym_lowercase+`
|`+Array+`
|A list of marker synonyms in lowercase for case-insensitive searches.

|`+text+`
|`+Array+`
|A general text field used for full-text search across multiple fields.

|`+auto_suggest+`
|`+Array+`
|A text field optimized for auto-suggestion functionality.

|===