|===
|Path|Type|Description

|`+allele_accession_id+`
|`+Array+`
|List of accession IDs for alleles.

|`+allele_name+`
|`+Array+`
|List of names for alleles.

|`+alt_mp_id+`
|`+Array+`
|List of alternative Mouse Phenotype (MP) IDs.

|`+annotated_higher_level_ma_term_name+`
|`+Array+`
|List of annotated higher level Mouse Anatomy (MA) term names.

|`+annotated_higher_level_mp_term_name+`
|`+Array+`
|List of annotated higher level Mouse Phenotype (MP) term names.

|`+assignment_status+`
|`+String+`
|The assignment status.

|`+ccds_id+`
|`+Array+`
|List of Consensus CDS (CCDS) IDs.

|`+child_ma_id+`
|`+Array+`
|List of child Mouse Anatomy (MA) IDs.

|`+child_ma_term+`
|`+Array+`
|List of child Mouse Anatomy (MA) terms.

|`+child_ma_term_synonym+`
|`+Array+`
|List of synonyms for child MA terms.

|`+child_mp_id+`
|`+Array+`
|List of child Mouse Phenotype (MP) IDs.

|`+child_mp_term+`
|`+Array+`
|List of child MP terms.

|`+child_mp_term_synonym+`
|`+Array+`
|List of synonyms for child MP terms.

|`+chr_end+`
|`+Number+`
|Chromosome end position.

|`+chr_name+`
|`+String+`
|Name of the chromosome.

|`+chr_start+`
|`+Number+`
|Chromosome start position.

|`+chr_strand+`
|`+String+`
|Chromosome strand.

|`+clan_acc+`
|`+Array+`
|List of clan accession numbers.

|`+clan_desc+`
|`+Array+`
|List of clan descriptions.

|`+clan_id+`
|`+Array+`
|List of clan IDs.

|`+conditional_allele_production_status+`
|`+String+`
|Production status of the conditional allele.

|`+crispr_allele_production_status+`
|`+String+`
|Production status of the CRISPR allele.

|`+data_type+`
|`+String+`
|The type of data.

|`+datasets_raw_data+`
|`+String+`
|Raw data from datasets.

|`+disease_alts+`
|`+Array+`
|List of alternative disease names.

|`+disease_classes+`
|`+Array+`
|List of disease classes.

|`+disease_human_phenotypes+`
|`+String+`
|Human phenotypes related to the disease.

|`+disease_id+`
|`+Array+`
|List of disease IDs.

|`+disease_source+`
|`+Array+`
|List of sources for disease data.

|`+disease_term+`
|`+Array+`
|List of disease terms.

|`+dmdd_image_data_available+`
|`+Boolean+`
|Indicates if DMDD image data is available.

|`+dmdd_lethal_data_available+`
|`+Boolean+`
|Indicates if DMDD lethal data is available.

|`+embryo_analysis_view_name+`
|`+String+`
|Name of the embryo analysis view.

|`+embryo_analysis_view_url+`
|`+String+`
|URL for the embryo analysis view.

|`+embryo_data_available+`
|`+Boolean+`
|Indicates if embryo data is available.

|`+embryo_modalities+`
|`+Array+`
|List of modalities for embryo analysis.

|`+ensembl_gene_id+`
|`+Array+`
|List of Ensembl gene IDs.

|`+es_cell_production_status+`
|`+String+`
|Production status of the ES cell.

|`+evid_code_rank+`
|`+Number+`
|Evidence code rank.

|`+exp_name+`
|`+Array+`
|List of experiment names.

|`+go_count+`
|`+Number+`
|Count of Gene Ontology (GO) terms associated.

|`+go_term_def+`
|`+Array+`
|List of GO term definitions.

|`+go_term_domain+`
|`+Array+`
|List of domains for GO terms.

|`+go_term_evid+`
|`+Array+`
|List of evidence for GO terms.

|`+go_term_id+`
|`+Array+`
|List of GO term IDs.

|`+go_term_name+`
|`+Array+`
|List of GO term names.

|`+go_uniprot+`
|`+Array+`
|List of UniProt IDs associated with GO terms.

|`+has_qc+`
|`+Number+`
|Indicates if quality control (QC) has been performed.

|`+hp_id+`
|`+Array+`
|List of Human Phenotype Ontology (HPO) IDs.

|`+hp_term+`
|`+Array+`
|List of HPO terms.

|`+human_curated+`
|`+Array+`
|Indicates if the gene has been curated by human experts.

|`+human_gene_symbol+`
|`+Array+`
|List of human gene symbols associated with the gene.

|`+human_symbol_synonym+`
|`+Array+`
|List of human gene Synonym associated with the gene.

|`+impc_novel_predicted_in_locus+`
|`+Array+`
|Indicates if novel IMPC predictions exist within the gene locus.

|`+impc_predicted+`
|`+Array+`
|Indicates if there are IMPC predictions for the gene.

|`+impc_predicted_in_locus+`
|`+String+`
|Specific locus for the IMPC predicted gene.

|`+impc_predicted_known_gene+`
|`+Array+`
|Indicates if known genes are predicted by IMPC within the locus.

|`+inferred_child_ma_id+`
|`+Array+`
|List of inferred child Mouse Anatomy (MA) IDs.

|`+inferred_child_ma_term+`
|`+Array+`
|List of inferred child MA terms.

|`+inferred_child_ma_term_synonym+`
|`+Array+`
|List of synonyms for the inferred child MA terms.

|`+inferred_ma_id+`
|`+Array+`
|List of inferred Mouse Anatomy (MA) IDs.

|`+inferred_ma_term+`
|`+Array+`
|List of inferred MA terms.

|`+inferred_ma_term_synonym+`
|`+Array+`
|List of synonyms for the inferred MA terms.

|`+inferred_selected_top_level_ma_id+`
|`+Array+`
|List of IDs for the inferred selected top-level MA terms.

|`+inferred_selected_top_level_ma_term+`
|`+Array+`
|List of inferred selected top-level MA terms.

|`+inferred_selected_top_level_ma_term_synonym+`
|`+Array+`
|List of synonyms for the inferred selected top-level MA terms.

|`+intermediate_mp_id+`
|`+Array+`
|List of intermediate Mouse Phenotype (MP) IDs.

|`+intermediate_mp_term+`
|`+Array+`
|List of intermediate MP terms.

|`+intermediate_mp_term_synonym+`
|`+Array+`
|List of synonyms for the intermediate MP terms.

|`+is_idg_gene+`
|`+Boolean+`
|Indicates if the gene is part of the IDG (Illuminating the Druggable Genome) project.

|`+is_umass_gene+`
|`+Boolean+`
|Indicates if the gene is associated with the University of Massachusetts.

|`+ma_id+`
|`+Array+`
|List of Mouse Anatomy (MA) IDs.

|`+ma_term+`
|`+Array+`
|List of MA terms.

|`+ma_term_definition+`
|`+Array+`
|List of definitions for the MA terms.

|`+ma_term_synonym+`
|`+Array+`
|List of synonyms for the MA terms.

|`+marker_name+`
|`+String+`
|General name of the marker.

|`+marker_symbol+`
|`+String+`
|Symbol representing the marker.

|`+marker_synonym+`
|`+Array+`
|List of synonyms for the marker.

|`+marker_type+`
|`+String+`
|Type of marker.

|`+mgi_accession_id+`
|`+String+`
|MGI (Mouse Genome Informatics) accession ID for the gene.

|`+mgi_novel_predicted_in_locus+`
|`+Array+`
|Indicates if novel MGI predictions exist within the gene locus.

|`+mgi_predicted+`
|`+Array+`
|Indicates if there are MGI predictions for the gene.

|`+mgi_predicted_in_locus+`
|`+String+`
|Specific locus for the MGI predicted gene.

|`+mgi_predicted_known_gene+`
|`+Array+`
|Indicates if known genes are predicted by MGI within the locus.

|`+mouse_curated+`
|`+Array+`
|Indicates if the gene has been curated by mouse experts.

|`+mouse_production_status+`
|`+String+`
|The production status of the mouse model for the gene.

|`+mp_id+`
|`+Array+`
|List of Mouse Phenotype (MP) IDs.

|`+mp_term+`
|`+Array+`
|List of MP terms.

|`+mp_term_definition+`
|`+Array+`
|List of definitions for the MP terms.

|`+mp_term_synonym+`
|`+Array+`
|List of synonyms for the MP terms.

|`+ncbi_id+`
|`+Array+`
|List of NCBI IDs associated with the gene.

|`+not_significant_top_level_mp_terms+`
|`+Array+`
|List of top-level MP terms not deemed significant.

|`+null_allele_production_status+`
|`+String+`
|Production status for null alleles.

|`+p_value+`
|`+Array+`
|List of p-values associated with statistical tests.

|`+parameter_name+`
|`+Array+`
|List of names for parameters used in experiments.

|`+parameter_stable_id+`
|`+Array+`
|List of stable IDs for the experimental parameters.

|`+pfama_acc+`
|`+Array+`
|List of Pfam A accession numbers.

|`+pfama_go_cat+`
|`+Array+`
|GO categories associated with Pfam A entries.

|`+pfama_go_id+`
|`+Array+`
|GO IDs associated with Pfam A entries.

|`+pfama_go_term+`
|`+Array+`
|GO terms associated with Pfam A entries.

|`+pfama_id+`
|`+Array+`
|List of Pfam A IDs.

|`+pfama_json+`
|`+Array+`
|JSON data related to Pfam A entries.

|`+phenotype_status+`
|`+String+`
|Status of the phenotyping effort for the gene.

|`+phenotyping_centre+`
|`+Array+`
|List of centres where phenotyping has been conducted.

|`+phenotyping_data_available+`
|`+Boolean+`
|Indicates if phenotyping data is available for the gene.

|`+pipeline_name+`
|`+Array+`
|List of names for pipelines used in data generation.

|`+pipeline_stable_id+`
|`+Array+`
|List of stable IDs for the pipelines.

|`+proc_param_name+`
|`+Array+`
|List of names for procedure parameters.

|`+proc_param_stable_id+`
|`+Array+`
|List of stable IDs for the procedure parameters.

|`+procedure_name+`
|`+Array+`
|List of names for procedures.

|`+procedure_stable_id+`
|`+Array+`
|List of stable IDs for the procedures.

|`+production_centre+`
|`+Array+`
|List of centres responsible for producing the models.

|`+scdb_id+`
|`+Array+`
|List of IDs from the Sanger Cancer Database.

|`+scdb_link+`
|`+Array+`
|List of links to the Sanger Cancer Database.

|`+selected_top_level_ma_id+`
|`+Array+`
|List of selected top-level Mouse Anatomy (MA) IDs.

|`+selected_top_level_ma_term+`
|`+Array+`
|List of selected top-level MA terms.

|`+selected_top_level_ma_term_id+`
|`+String+`
|ID of the selected top-level MA term.

|`+selected_top_level_ma_term_synonym+`
|`+Array+`
|List of synonyms for the selected top-level MA terms.

|`+seq_region_end+`
|`+Number+`
|End position on the sequence region.

|`+seq_region_id+`
|`+String+`
|ID of the sequence region.

|`+seq_region_start+`
|`+Number+`
|Start position on the sequence region.

|`+significant_top_level_mp_terms+`
|`+Array+`
|List of significant top-level MP terms.

|`+subtype+`
|`+Array+`
|List of subtypes associated with the gene.

|`+top_level_mp_definition+`
|`+String+`
|Definition of the top-level MP term.

|`+top_level_mp_id+`
|`+Array+`
|List of top-level MP IDs.

|`+top_level_mp_term+`
|`+Array+`
|List of top-level MP terms.

|`+top_level_mp_term_synonym+`
|`+Array+`
|List of synonyms for the top-level MP terms.

|`+type+`
|`+String+`
|The type of document or data.

|`+vega_id+`
|`+Array+`
|List of VEGA IDs associated with the gene.

|`+xref+`
|`+Array+`
|List of cross-references for the gene.

|===